src/vma/sock/sockinfo.cpp

```diff
@@ -899,7 +907,7 @@ void sockinfo::rx_add_ring_cb(flow_tuple_with_local_if &flow_key, ring* p_ring,
 		rx_ring_iter->second->refcnt++;
 	}
 
-#ifdef DEFINED_VMAPOLL
+#if defined(DEFINED_VMAPOLL) || defined(HAVE_MP_RQ)
```

Comment:
Please verify striding rq doesn't break VMAPOLL.

#### Introduced in:
`add cyclic function api and enalbe related functions when not using VMA_POLL`